### PR TITLE
feat: add intro

### PR DESCRIPTION
**Pull request by danielsp45**
image

### CODE REVIEW

**Review by ruilopesm — APPROVED** [review]

File: src/pages/index.astro [hunk 1]
```diff
@@ -35,12 +35,12 @@ const description = "The best place to grow ideas from the ground";
...
-    <!--   <Features /> -->
-    <!--   <Compatibility /> -->
-    <!--   <Showcase /> -->
-    <!-- </div> -->
-    <!-- <Footer /> -->
+    <div class="space-y-24 px-8 py-32">
+      <Intro />
+      <!-- <Features /> -->
+      <!-- <Compatibility /> -->
+      <!-- <Showcase /> -->
```
Comment (MarioRodrigues10) [inline src/pages/index.astro:42]:
```suggestion

```

**Review by MarioRodrigues10 — DISMISSED** [review]

File: src/components/intro.astro [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
+        AND OTHER ABILITIES THAT ARE CRUCIAL TO THE COMPLETION OF A SUCCESSFUL
+        PROJECT - TEAMWORK, COMMUNICATION, TIME MANAGEMENT, CREATIVITY AND
+        CRITICAL THINKING.
+      </p>
+    </div>
+    <div>
+      <p>
+        AN ENVIRONMENT OF ACTIVE LEARNING, MENTORING AND NETWORKING WILL BE
+        FOSTERED, GIVING ALL PARTICIPANTS THE OPPORTUNITY TO MAKE THE NEXT MOVE
+        IN THEIR PROFESSIONAL AND ACADEMIC CAREERS.
```
Comment (joaodiaslobo) [inline src/components/intro.astro:31]:
```suggestion
    class="mt-10 grid max-w-xl grid-cols-1 gap-8 text-xl leading-7 text-justify lg:max-w-6xl lg:grid-cols-2 uppercase"
  >
    <div>
      <p>
        This competition not only seeks to enhance the technical skills and
        expertise of its participants, but also to hone a variety of soft skills
        and other abilities that are crucial to the completion of a successful
        project - teamwork, communication, time management, creativity and
        critical thinking.
      </p>
    </div>
    <div>
      <p>
        An environment of active learning, mentoring and networking will be
        fostered, giving all participants the opportunity to make the next move
        in their professional and academic careers.
```

File: src/components/intro.astro [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
-    performance meets <span class="text-primary">powerful</span> developer experience.
+<ContentSection title="What is it?" id="intro">
+  <Fragment slot="lead" class="tracking-wide">
+    FOR <span class="text-primary">48 HOURS</span>, OVER <span
+      class="text-primary">THREE DAYS</span
+    >, PARTICIPANTS WILL WORK AS TEAMS TO CREATE A <span class="text-primary"
+      >TECH PROJECT</span
+    > CENTRED AROUND AN ENTREPRENEURIAL AND IMPACTFUL SOLUTION THAT TACKLES ONE OF
+    THE PROPOSED CHALLENGES.
   </Fragment>
```
Comment (joaodiaslobo) [inline src/components/intro.astro:13]:
```suggestion
  <Fragment slot="lead" class="tracking-wide uppercase">
    For <span class="text-primary">48 hours</span>, over <span
      class="text-primary">three days</span
    >, Participants will work as teams to create a <span class="text-primary"
      >tech project</span
    > Centred around an entrepreneurial and impactful solution that tackles one of
    the proposed challenges.
  </Fragment>
```

**Review by joaodiaslobo — CHANGES_REQUESTED** [review]
For better accessibility and code readability use Tailwind's uppercase class instead of manually writing the text in uppercase

File: src/pages/index.astro
Comment (MarioRodrigues10) [inline src/pages/index.astro:1]: Remove unnecessary imports

**Review by MarioRodrigues10 — COMMENTED** [review]
Please also remove unnecessary node modules folder

**Review by MarioRodrigues10 — APPROVED** [review]
💚

**Pull request merged**